API_GUIDE.md

```diff
@@ -135,11 +135,19 @@ if __name__ == '__main__':
 ```
 
 There are three differences between this multi-device snippet and the previous
-single device snippet:
-
-- `xmp.spawn()` creates the processes that each run an XLA device.
-- `MpDeviceLoader` loads the training data onto each device.
-- `xm.optimizer_step(optimizer)` consolidates the gradients between cores and issues the XLA device step computation.
+single device snippet. Let's go over then one by one.
+
+- `xmp.spawn()`
+  - Creates the processes that each run an XLA device
+  - Each process will only be able to access the device assigned to the current process. For example on a TPU v4-8, there will be 4 processes being spawn up and each process will own a TPU device.
```

Comment:
By "TPU device", do you mean a TPU chip (containing 2 cores)?

Also, just checking if it's on a v3-8, would there be 8 processes instead?

Reply:
I felt like chip is too much of a detail. On V4-8 there are 4 devices, each of them is a chip. One v3-8 there are 8 devices, each of them is a core.